src/quo/components/avatars/dapp_avatar/view.cljs

```diff
@@ -0,0 +1,31 @@
+(ns quo.components.avatars.dapp-avatar.view
+  (:require [quo.components.avatars.dapp-avatar.schema :as dapp-avatar.schema]
+            [quo.components.avatars.dapp-avatar.style :as style]
+            [react-native.core :as rn]
+            [react-native.hole-view :as hole-view]
+            [schema.core :as schema]))
+
+(defn view-internal
+  [{:keys [context? image network-image container-style]}]
+  [rn/view
+   {:style               (merge style/container container-style)
+    :accessibility-label :dapp-avatar}
+   [hole-view/hole-view
+    {:key   (hash context?)
```

Comment:
Same as here https://github.com/status-im/status-mobile/pull/20141#discussion_r1609847148, wdyt?

Reply:
@ajayesivan isn't it enough if we only provide `context?` or maybe booleans arent a valid react key?

Reply:
@briansztamfater Changed the code to only add `key` prop for Android

Reply:
@ulisesmac Yes we can use `boolean` value for `:key`. I have updated the code.